samples/unit-tests/series-bubble/minmaxsize/demo.js

```diff
@@ -1,3 +1,52 @@
+QUnit.test('Setting X axis extremes on bubble (#5167)', function (assert) {
```

Comment:
Not related to `minSize` or `maxSize`, so a better location would be `unit-tests/series-bubble/extremes`.